### PR TITLE
(VDB-1594) Additional Bite/Flip integration tests

### PR DESCRIPTION
**Pull request by rmulhol**

### CODE REVIEW

**Comment by rmulhol** [comment]
This is the first incremental chunk in what will likely be an ongoing effort to get caught up on integration tests. Marking it for review now because (1) there are some higher priority things that need attention right now, and (2) don't want to bloat the diff too much

File: transformers/integration_tests/bite_test.go [hunk 1]
```diff
@@ -32,18 +30,46 @@ import (
...
+		urn := "0x0A051CD913dFD1820dbf87a9bf62B04A129F88A5"
+		ilk := "0x4554482d41000000000000000000000000000000000000000000000000000000"
+		expectedResult := biteModel{
+			Ink:  "50000000000000000000",
+			Art:  "4460522851157616216837",
+			Tab:  "4466031366353941646208178591268931635087392443453",
+			Flip: test_data.FlipEthV100Address(),
+			Id:   "112",
+		}
+		biteIntegrationTest(blockNumber, test_data.Cat100Address(), constants.Cat100ABI(), ilk, urn, expectedResult)
```
Comment (yaoandrew) [inline transformers/integration_tests/bite_test.go:44]: 💯

File: transformers/integration_tests/flip_kick_test.go [hunk 2]
```diff
@@ -29,65 +29,225 @@ import (
...
+			Usr:   "0x2A4ed588c523Cf7F8003c83419d8cEfd18eE599B",
+			Gal:   "0xA950524441892A31ebddF91d3cEEFa04Bf454466",
+		}
+		flipKickIntegrationTest(10570493, test_data.FlipZrxAV109Address(), constants.FlipV100ABI(), expected)
+	})
+
+	// TODO: add MCD_FLIP_ZRX_A_1.1.0 test when available
+})
+
+func flipKickIntegrationTest(blockNumber int64, contractAddressHex, contractABI string, expectedResult flipKickModel) {
```
Comment (yaoandrew) [inline transformers/integration_tests/flip_kick_test.go:200]: 🥇 Nice.. I like the way this looks.

**Review by yaoandrew — APPROVED** [review]
LGTM 🚢

**Pull request merged**